src/Settings.php

```diff
@@ -127,7 +127,7 @@ public static function parseArguments(array $arguments)
         }
 
         foreach ($arguments as $argument) {
-            if ($argument{0} !== '-') {
+            if (strpos($argument, '-') !== 0) {
```

Comment:
Maybe you could just use `$argument[0]`, instead of using `strpos`.

Reply:
Indeed, I just pushed an edit on my branch